README.md

```diff
@@ -53,35 +53,19 @@ The diagram below shows that the ATD lab topology consists of two Spines, four L
     git config --global user.name "Your Name"
     ```
 
-2. Run the install script to clone the required repositories and install any dependencies
+2. Set credentials and install any required tools
 
     ```shell
-    curl -fsSL https://get.avd.sh/atd/install.sh | sh
+    cd /home/coder/project/labfiles
+    export LABPASSPHRASE=`cat /home/coder/.config/code-server/config.yaml| grep "password:" | awk '{print $2}'`
+    ansible-galaxy collection install -r requirements.yml
+    export ARISTA_AVD_DIR=$(ansible-galaxy collection list arista.avd --format yaml | head -1 | cut -d: -f1)
+    pip3 install -r ${ARISTA_AVD_DIR}/arista/avd/requirements.txt
+    git clone https://github.com/arista-netdevops-community/atd-avd.git
+    cd atd-avd
```

Comment:
LGTM in general, but:
- I'd make Ansible install more explicit: `ansible-galaxy collection install arista.avd:==4.1.0`. Hiding that behind requirements YAML makes it more confusing. requirements.yml can be deleted in that case. And there is no need to install ansible.cvp separately as it will come with ansible.avd.
- We have to force install ansible core to be sure that it's controlled by atd-avd repo and not by ATD coder container. Something like:
 
```bash
pip install --upgrade pip
pip3 install --ignore-installed "ansible-core>=2.13.1,<2.14.0"
```
- Based on SE feedback we need a single line install script long term. But not the current one, as it's not working correctly. Could be step 2 as well.